tests/_fbe.pm

```diff
@@ -59,6 +59,11 @@ sub run {
     # desktop should be visible
     check_desktop_clean();
     save_screenshot();
+
+    # Check there are no remaining processes owned by gnome-initial-setup
+    $self->root_console(tty=>4);
```

Comment:
Doesn't work because `$self` is not a subclass of `installedtest`. Hm.

Reply:
I missed that in my review. Thinking…

Reply:
Maybe do something like `_enable_coredumps.pm`?